### ISSUE TITLE
Footer. Top of page link target does not exist

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#451]
## Description of issue

Describe what is failing
- We have a top of page link targeting #top, but we don't have a target with that id. 

## Potential fixes

- Change the href to '#'

or

-  Add the missing target somewhere in the header. Think about how the headers are responsive and make sure the target is always focusable and visible. 

## Resources

- https://www.w3.org/TR/WCAG21/#bypass-blocks
- - found with [Arc toolkit](https://chrome.google.com/webstore/detail/arc-toolkit/chdkkkccnlfncngelccgbgfmjebmkmce?hl=en)
